controllers/platformoperator_controller.go

```diff
@@ -102,10 +109,51 @@ func (r *PlatformOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	u := updater.NewPlatformOperatorUpdater(r.Client)
+	defer func() {
+		if err := u.Apply(ctx, po); err != nil {
+			log.Error(err, "failed to update status")
+		}
+	}()
+
+	u.UpdateStatus(
+		updater.SetPhase(platformopenshiftiov1alpha1.PhaseFailing),
```

Comment:
One additional follow-up question: why do we set the phase to failing here initially? are we expecting to overwrite this status change with later status funcs?

Reply:
Another Copy/Paste artifact. It mean for this to be `PhaseInstalling`. Updated.